src/AppBundle/Controller/Pa/PaSettingsController.php

```diff
@@ -81,4 +81,33 @@ public function profileEditAction(Request $request)
             'form' => $form->createView()
         ];
     }
+
+    /**
+     * Display the password edit page
+     *
+     * @Route("/your-details/change-password", name="pa_profile_password_edit")
+     * @Template("AppBundle:Pa/Settings:profilePasswordEdit.html.twig")
+     **/
+    public function profilePasswordEditAction(Request $request)
```

Comment:
this is the same as the existing UserController::passwordEditAction(), except for the route and template.

I'd try to implement it by linking to the existing controller action, rendering the existing template, and just put some if/else (both controller and twig) based on what the user is (app.user.isDeputyPa() in twig, and $this->GetUser()->isDeputyPA() in the controller). What do you think ?